bears/vcs/git/GitCommitBear.py

```diff
@@ -104,8 +104,19 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
 
         stdout = stdout.rstrip('\n')
         pos = stdout.find('\n')
+        count = 0
         shortlog = stdout[:pos] if pos != -1 else stdout
-        body = stdout[pos+1:] if pos != -1 else ''
+        pos = pos + 1
+
+        while (stdout[pos+1] == '\n'):
+            count++
+
+        if count == 0:
+            body = stdout[pos+1:] if pos != -1 else ''
+        else
+            yield Result(self,
```

Comment:
Also I think this `if` logic should be executed only after the commit message is non-empty, i.e. move your logic below this code
```
if len(stdout) == 0:
            if not allow_empty_commit_message:
                yield Result(self, 'HEAD commit has no message.')
return
```